tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -792,10 +783,11 @@ def mt_full_interface
 # found
 def fabricpath_interface
   # Search for F2E/F3 cards on device, create an interface name if found
-  cmd = get_vshell_cmd('sh mod')
-  out = on(agent, cmd, pty: true).stdout[/^(\d+)\s.*N7K-F(?:2.*25E|3)/]
-  slot = out.nil? ? nil : Regexp.last_match[1]
-  "ethernet#{slot}/1" unless slot.nil?
+  cmd = '-p cisco.feature_compatible_module_iflist.fabricpath'
+  if_array_str = on(agent, facter_cmd(cmd)).stdout.chomp
+  if_array_str.gsub!(/[\[\]\n\s"]/, '')
+  if_array = if_array_str.split(',')
+  if_array[0] unless if_array.empty?
 end
```

Comment:
For post-release, consider changing this method from `fabricpath_interface` to `compatible_interface(feature)` since we'll probably need this for other linecard-specific providers.